lib/absinthe/phase/document/complexity.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Absinthe.Phase.Document.Complexity do
+  @moduledoc false
+
+  # Analyses document complexity.
+
+  alias Absinthe.{Blueprint, Phase}
+
+  use Absinthe.Phase
+
+  @default_complexity 1
```

Comment:
Returning a literal is slightly faster than doing a Map.get but honestly both a literal and a Map.get are fast enough that the resolution complexity is less than the overhead of resolving the field in general. I feel like 1 is a good floor, I can't think of any reason to go less.